node/action/base.go

```diff
@@ -58,6 +58,11 @@ type Transaction interface {
 	Resolve(interface{}, Commands)
 }
 
+type SignedTransaction struct {
+	Transaction
+	signature []byte
```

Comment:
If you need to access this field outside the `action` package you need to capitalize it.

Reply:
@reblws , sorry, what should I do?

Reply:
Do you mean, I need to use Signature name?

Reply:
Yes, in Go lowercase means private, while Uppercase means public.